src-backbone/app/locales/en/translation.json

```diff
@@ -103,13 +103,13 @@
     "MIME Type": "MIME Type",
     "Default Answer": "Default Answer",
     "DATE": "Date",
-    "ENTRY": "Entry",
-    "SELECT": "Selection",
+    "ENTRY": "Text Entry",
+    "SELECT": "Dropdown",
     "MULTI_SELECT": "Multiple Choice",
     "RADIO": "Single Choice",
     "PICTURE": "Picture",
-    "PLUGIN": "Plugin (file)",
-    "ENTRY_PLUGIN": "Plugin (entry)",
+    "PLUGIN": "Plugin",
+    "ENTRY_PLUGIN": "Entry Plugin",
```

Comment:
I literally have no idea what this actually is in the app. I'd rather leave it as is for now.